### ISSUE TITLE
Exclude application windows from being restored

### ISSUE THREAD

**Issue by braintapper** [kangyu-california/PersistentWindows#90]
Just a feature request.

It would be nice if there was an exclusion list (either in settings or a text file) to exclude certain applications from having their window positions tracked in case they don't play nicely with PersistentWindows.

For example, my Skype window's last position is restored even if the window wasn't open prior to the monitors (or system) sleeping. I think it would be handy to be able to exclude `skype.exe`'s windows from being tracked in this case.

**Comment by kangyu-california** [comment]
The problem might be caused by failure to capture/restore a minimized window, which should have been fixed by release 5.18 and 5.19

if any window still does not restore correctly, you may exclude it from auto restore by holding Ctrl+Shift keys then move it.

**Comment by 3941** [comment]
Is there a file I can edit to exclude a window?
MPV is my media player, and it usually takes on the size of whatever video I play, which is what I like it to do.
I CAN NOT move its window while holding ctrl+shift, so I cannot exclude it that way.

**Comment by kangyu-california** [comment]
@3941 
run PW with command line option, for example
-ignore_process "mpv.exe"

**Comment by 3941** [comment]
Ah, thank you very much. I don't know how I overlooked that one. I guess I didn't figure it would be a command line parameter.
Have added it to the task in task scheduler and it works fine!

I've also tried the ctrl+close function of the new version, but although that actually worked, by which I mean I was able to close MPV while holding control, it had no effect on PW either. I guess the MPV window is just weird. I use it totally borderless with controls that only show up on mouseover.

Anyways, problem solved, thanks again! ❤
